### PR TITLE
lesson-1.8

### PR DESCRIPTION
**Pull request by mjs94080**

### CODE REVIEW

File: my-react-app/src/TodoList.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ function TodoList({ todoList, onRemoveTodo }) {
           <TodoListItem
             key={item.id}
             id={item.id}
-            title={item.title}
+            title={item.fields.Name}
```
Comment (Asel-K) [inline my-react-app/src/TodoList.js:12]: I'd suggest to move your updated todo item ({item.fields.Name}) from TodoList.js to TodoListItem.js as it was asked in the instructions.

File: my-react-app/src/TodoListItem.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import React from "react";
+import { findRenderedDOMComponentWithClass } from "react-dom/test-utils";
+import TodoList from "./TodoList";
 
 function TodoListItem({ title, onRemoveTodo, id }) {
   return (
```
Comment (Asel-K) [inline my-react-app/src/TodoListItem.js:6]: The updated todo item title goes to this file according to the instructions ( line 8)

```
(Open src/TodoListItem.js
 Update the todo item title to reference the new object format (hint: todo.fields.Title) 
```

**Review by Asel-K — COMMENTED** [review]
Looks good! Just make one change as asked in comments. You may ask for assistance during the mentor sessions or 1:1 mentor sessions if needed.

You may include your changes to your next pull request and don't need to resubmit this PR. As  per your question about sorting, we will make those edits in our future assignments. You may take a look at Lesson 5.1.

File: my-react-app/src/TodoListItem.js (hunk 2 above)
Comment (mjs94080) [reply my-react-app/src/TodoListItem.js:6]: I did what you suggest and now my browser is broke.

This completely broke and I cant get it back!!!!!!!!!!!!!!!!!!!

Help me on this, you suggested this.

Comment (mjs94080) [reply my-react-app/src/TodoListItem.js:6]: :\Users\admin\Documents\Github\react-bald-eagle\my-react-app\src'       
WARNING in [eslint]
src\TodoListItem.js
  Line 2:10:  'findRenderedDOMComponentWithClass' is defined but never used  no-unused-vars
  Line 3:10:  'isHtmlElement' is defined but never used
     no-unused-vars
  Line 4:8:   'TodoList' is defined but never used
     no-unused-vars

ERROR in ./src/TodoListItem.js 6:0-58
Module not found: Error: Can't resolve 'react-router-dom/dist/dom' in 'C:\Users\admin\Documents\Github\react-bald-eagle\my-react-app\src'

Comment (mjs94080) [reply my-react-app/src/TodoListItem.js:6]: this is the result of your suggestion and now I cant get it back to work!

give me the solution PLEASE!

**Pull request merged**